prow/cluster/boskos.yaml

```diff
@@ -98,7 +95,7 @@ metadata:
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: crd-creator
+  name: boskos
```

Comment:
Someone needs to manually delete the old `crd-creator` role after this gets deployed

Reply:
@BenTheElder is the current on-call

Reply:
presumably after the next bump? I've had a full schedule today and haven't bumped yet.

Reply:
This has been deployed now, so the old role binding (named `crd-creator`) can be deleted.

Reply:
this is done